pytket/tests/qasm_test.py

```diff
@@ -955,6 +956,49 @@ def test_const_condition() -> None:
     )
 
 
+def test_conditional_multi_line_ops() -> None:
+    # https://github.com/CQCL/tket/issues/1491
+    c = Circuit(0, 3)
+    c.add_c_setbits(values=[True, True], args=[Bit(1), Bit(2)], condition=Bit(0))
```

Comment:
I see the following problem with this solution, what happens if the bit of the condition is used later on?
```
c.add_c_setbits(values=[False, True], args=[Bit(0), Bit(2)], condition=Bit(0))
```
When the conversion for this example works in the same way, this could run the first line, but not the second?
Do you know what I mean?